src/shared/tw.tsx

```diff
@@ -1,7 +1,11 @@
+import { z } from "zod";
+import { formSchema } from "./onboarding";
 type Optional<T> = T | null | undefined;
 
 export function classes(...classes: Optional<string>[]) {
   return classes.filter(Boolean).join(" ");
 }
 
+export type TypeFormSchema = z.infer<typeof formSchema>;
```

Comment:
IMO, `type Type...` is not a good name, as we already said it's a `type` and it's also using pascal case, which is a convention for naming types.

maybe, `FormSchema` is good enough

Reply:
This is also only related to form types, so it should not be placed inside this tailwind utils